### ISSUE TITLE
Bottom control bar UI suggestion

### ISSUE THREAD

**Issue by nekrasovdmitriy** [Pullusb/storytools#46]
Suggestion is to group buttons in 3 categories, add color coding, follow one logic (key button to the right side of the each group).

As an option, add group names above.

![ui](https://github.com/user-attachments/assets/1dcfe789-5691-42d0-8e9a-88a4275a9641)

**Comment by Pullusb** [comment]
Hi again ! Thanks for the input and taking the time to create a mockup

### icon placements
I thought a lot about the placement of the new GP icons. First, I've added them in the same line (here was the first test):
![image](https://github.com/user-attachments/assets/d9fefbf8-6b56-4dcc-9a3f-1ab5b95922af)

But after working on it, there is a good reason for going in a second line entirely: Context.
All the other tools can be used with any object in any context, not only in GP context. So The buttons are always visible (that's also why I did not group the "Go draw" button with the keys, this one is also usable out of GP objects)
Considering that, the buttons disappear when the active object is not a GP, and I don't want the bottom bar to resize when the switch happens.

That said, the position is not set in stone. I consider going for a vertical lateral viewport placement, but other priorities now.

For the record. I also tested centering the `gp bar` above the icons, but it did not felt nice, and I am satisfied with the honeycomb mode for now.

![image](https://github.com/user-attachments/assets/44ef31f0-f6de-4e3a-ad22-515a86e16bbe)

### About Colors
In the firsts addon iterations, there was a color code. But the users asked me to go back to a more "blended" design ;). It ended up being too distracting, and I agree.
There was an old tweet about that :  https://x.com/pullusb/status/1662023923465068545

![image](https://github.com/user-attachments/assets/22f85b79-9b08-49ca-99b9-596cfa8f5ed6)

That's why it's only shades of grey and will stay that way. But I agree the new button could be using a different shade. I plan to add another UI custom color in preference for those, and probably use a darker tone than object by default. Maybe the same as camera controls, since there are separated by distance, and I'm not fond of multiplying color... will have to be tested as usual.


### Lock orbit navigation
It's interesting to see that you'd group the `lock to pan view` in the camera category. Though it's technically not related to the camera, it affects the navigation and thus the view, so it can see this placement working. Not sure yet

### Placement bug
I see there is a position issue on your screenshot !
![image](https://github.com/user-attachments/assets/5730b77a-b51e-4e77-88db-124a27fd7171)

I need to fix this. In the meantime, could you tell me what is your monitor resolution and if you're using resolution scaling different above or below `1.0` ? (in preferences > Interface > display > resolution scale)

### About feedbacks
I suppose hearing "it will stay that way" can be frustrating, that's why I justified the choices a lot here (sometimes I don't 😅).
In any case, just know that I never reject incoming ideas right away, and even when I choose to ignore some, it opens doors to think about it again and, with other people feedbacks, it can be reconsidered in the future.
So basically, if I say no, please never take it personally 😄  
Thanks again for the feedback.

I'll leave the ticket open until the position bug is solved.

Cheers

**Comment by nekrasovdmitriy** [comment]
> could you tell me what is your monitor resolution and if you're using resolution scaling different above or below 1.0 ? (in preferences > Interface > display > resolution scale)

i have 4k display 3840 x 2160 and macos UI scaled to  2560 x 1440

**Comment by nekrasovdmitriy** [comment]
"Duplicate key" icon suggestion 
![image](https://github.com/user-attachments/assets/f663d6ab-f2ea-4418-b3bc-cda0f5fd895d)

**Comment by Pullusb** [comment]
I'm well aware that the icon is not ideal, but unfortunately it's not possible to use custom icons with gizmo buttons.
You can only add images in handmade openGL UI that are complex to make. For now I prefer to stay with native gizmo buttons, those are practical and efficient. Custom UI are for later (if at all)

In short. I'm limited to Blender's native icon set :

![image](https://github.com/user-attachments/assets/943fa7d4-9690-47d1-b6bc-6f651a2ec7c0)

**Comment by Pullusb** [comment]
Bottom bar UI should be fixed in latest version (GP related buttons have now their own color code in preferences).

Closing the ticket. Reopen if you still have weird offsets on gizmo buttons.
